private-compressed-account/package.json

```diff
@@ -11,6 +11,7 @@
     "circomlib": "^2.0.5",
     "circomlibjs": "^0.1.7",
     "@lightprotocol/zk.js": "0.3.2-alpha.14",
+    "@lightprotocol/cli": "0.3.2-alpha.20",
```

Comment:
the most current version is 21

Reply:
```suggestion
    "@lightprotocol/cli": "0.3.2-alpha.21",
```

Reply:
yes, but it's broken 

$ light test-validator -b && yarn ts-node src/main.ts --exit

⠧ Performing setup tasks...

Performing setup tasks...

📥 Downloading https://github.com/Lightprotocol/light-protocol/releases/download/v0.3.2/spl_noop.so...
 ████████████████████████████████████████ 100% | ETA: 0s | 19960/19960
📥 Downloading https://github.com/Lightprotocol/light-protocol/releases/download/v0.3.2/merkle_tree_program.so...
 ████████████████████████████████████████ 100% | ETA: 0s | 739072/739072
📥 Downloading https://github.com/Lightprotocol/light-protocol/releases/download/v0.3.2/verifier_program_zero.so...
 ████████████████████████████████████████ 100% | ETA: 0s | 483824/483824
📥 Downloading https://github.com/Lightprotocol/light-protocol/releases/download/v0.3.2/verifier_program_storage.so...
 ████████████████████████████████████████ 100% | ETA: 0s | 516696/516696
📥 Downloading https://github.com/Lightprotocol/light-protocol/releases/download/v0.3.2/verifier_program_one.so...
 ████████████████████████████████████████ 100% | ETA: 0s | 533240/533240
📥 Downloading https://github.com/Lightprotocol/light-protocol/releases/download/v0.3.2/verifier_program_two.so...
 ████████████████████████████████████████ 100% | ETA: 0s | 492032/492032
Executing command killall...
No matching processes belonging to you were found
killall exited with code 1
Executing command killall...
No matching processes belonging to you were found
killall exited with code 1
error Command failed with exit code 1.

Reply:
is there another error or is it an infinite loop?